src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,41 @@
 package core.basesyntax;
 
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        StringBuilder result = new StringBuilder();
+        result.append("Report for period ").append(dateFrom).append(" - ").append(dateTo)
+                .append(System.lineSeparator());
+        try {
+            Date fromDate = new SimpleDateFormat("dd.MM.yyyy").parse(dateFrom);
+            Date toDate = new SimpleDateFormat("dd.MM.yyyy").parse(dateTo);
```

Comment:
don't wrap whole method in try-catch, better use DateTimeFormatter.ofPattern which doesn't throw exception